--- etc/pacman.conf	2016-03-04 12:22:33.588589946 +0100
+++ etc/pacman.conf	2016-03-04 12:44:21.197561372 +0100
@@ -37,8 +37,8 @@
 #VerbosePkgLists
 
 # PGP signature checking
-#SigLevel = Optional
-#LocalFileSigLevel = Optional
+SigLevel = Required DatabaseOptional
+LocalFileSigLevel = Optional
 #RemoteFileSigLevel = Optional
 
 #
@@ -60,6 +60,15 @@
 # uncommented to enable the repo.
 #
 
+[core]
+Include = /etc/pacman.d/mirrorlist
+
+[extra]
+Include = /etc/pacman.d/mirrorlist
+
+[community]
+Include = /etc/pacman.d/mirrorlist
+
 # An example of a disabled remote package repository with multiple servers
 # available. To enable, uncomment the following lines. You can add preferred
 # servers immediately after the header and they will be used before the
